src/sub.c

```diff
@@ -301,7 +310,7 @@ natsSub_deliverMsgs(void *arg)
     }
 
     natsSub_Lock(sub);
-    onCompleteCB        = sub->onCompleteCB;
```

Comment:
That is an example of what I was talking about :-)

Reply:
I am not sure I am following :) I _think_ this one is VERY low overhead since it's done once, so kind of does not matter that we re-lock here?

Reply:
Oh no, I was just saying that I was aligning the `= ...` with the line below, that was a comment about "tabulation", that's all.

Reply:
@kozlovic do you mind emailing me a copy of your VSCode `settings.json` so I can sync the auto-formatting?